### PR TITLE
Code cleanup of the Backpack module.

### PR DESCRIPTION
**Pull request by jokoho48**
Code cleanup of the Backpack module.

Split PRs from #1989.

### CODE REVIEW

File: addons/backpacks/functions/fnc_backpackOpened.sqf [hunk 1]
```diff
@@ -29,8 +28,8 @@ _sounds = [
     "a3\sounds_f\characters\ingame\AinvPpneMstpSlayWpstDnon_medic.wss",
     "a3\sounds_f\characters\ingame\AinvPpneMstpSlayWrflDnon_medic.wss"*/
 
-    QUOTE(PATHTO_R(sounds\zip_in.wav)),
-    QUOTE(PATHTO_R(sounds\zip_out.wav))
+    QUOTE(PATHTOF(sounds\zip_in.wav)),
```
Comment (PabstMirror) [inline addons/backpacks/functions/fnc_backpackOpened.sqf]: These are not equivalent
"z\ace\addons\backpacks\sounds\zip_out.wav" vs "\z\ace\addons\backpacks\sounds\zip_out.wav"

`say3d` is very picky and won't accept PATHTOF

File: addons/backpacks/functions/fnc_getBackpackAssignedUnit.sqf [hunk 2]
```diff
@@ -2,21 +2,20 @@
...
-
-_unit = objNull;
+params ["_unit","_backpack"];
+_target = objNull;
 {
-    if (backpackContainer _x == _backpack) exitWith {_unit = _x};
-} forEach (allUnits + allDeadMen);
-_unit
+    if (backpackContainer _x == _backpack) then {_target = _x; breakTo "main"};
+} forEach nearestObjects [_unit, ["Man"],2];
```
Comment (PabstMirror) [inline addons/backpacks/functions/fnc_getBackpackAssignedUnit.sqf]: This probably is faster, but I managed to open a backpack at 2.8 meters

Comment (jokoho48) [inline addons/backpacks/functions/fnc_getBackpackAssignedUnit.sqf]: ok i can increase this

File: addons/backpacks/functions/fnc_backpackOpened.sqf (hunk 1 above)
Comment (jokoho48) [inline addons/backpacks/functions/fnc_backpackOpened.sqf]: ok that is interessting thanks that i dont know

**Comment by Glowbal** [comment]
This should be fine pending changes as per comments from @PabstMirror .

**Comment by jokoho48** [comment]
ok this is Done

**Pull request merged**